content/ADR-76-urn-prefix-search.md

```diff
@@ -27,9 +27,32 @@ The decision is to solve this at Content Server Level as the approach of the fil
 
 The creation of a [new endpoint in the Content Server](https://github.com/decentraland/catalyst-api-specs/pull/21):
 
-`GET /entities/currently-pointed/{urnPrefix}`
+#### REQUEST
 
-That returns a list with all active entityIds that point to at least one pointer with the urnPrefix.
+`GET /content/entities/active/collections/{collectionUrn}`
+
+#### RESPONSE
+
+```json
+[
+  {
+    "entityId": "bafkreidtudd3elgkwojklsfr7w4rrrnswcsnwexpchy4raqnvfwiid52mm",
+    "pointer": "urn:decentraland:matic:collections-thirdparty:woodies:0x134460d32fc66a6d84487c20dcd9fdcf92316017:0"
+  },
+  {
+    "entityId": "bafkreiendurkhozosr2np6vswta745jjnbvym65oiow5yqss4eed2oanuy",
+    "pointer": "urn:decentraland:matic:collections-thirdparty:woodies:0x134460d32fc66a6d84487c20dcd9fdcf92316017:1"
+  }
+]
+```
+
+That returns a list with all active entityIds whose urn are part of the `collectionUrn` given.
+
+The `collectionUrn` format must follow the one defined in [URN Repository](https://github.com/decentraland/urn-resolver#:~:text=supported%20on%20polygon-,decentraland%3A%7Bprotocol%7D%3Acollections%2Dthirdparty%3A%7BthirdPartyName%7D%3A%7BcollectionId%7D,-%3A%20Resolves%20the%20ethereum) which is: `decentraland:{protocol}:collections-thirdparty:{thirdPartyName}:{collectionId}:{itemId}`.
```

Comment:
Does it include `itemId`? Or does it stop at `collectionId`?

Reply:
It can be any of: 
- `decentraland:{protocol}:collections-thirdparty:{thirdPartyName}`
- `decentraland:{protocol}:collections-thirdparty:{thirdPartyName}:{collectionId}`
- `decentraland:{protocol}:collections-thirdparty:{thirdPartyName}:{collectionId}:{itemId}`

Is it clear?